modules/renter/contractor/uptime.go

```diff
@@ -0,0 +1,94 @@
+package contractor
+
+import (
+	"time"
+
+	"github.com/NebulousLabs/Sia/build"
+	"github.com/NebulousLabs/Sia/modules"
+)
+
+// uptimeInterval specifies how frequently hosts are checked.
+var (
+	uptimeInterval = func() time.Duration {
+		switch build.Release {
+		case "dev":
+			return 1 * time.Minute
+		case "standard":
+			return 30 * time.Minute
+		case "testing":
+			return 100 * time.Millisecond
+		}
+		panic("undefined uptimeInterval")
+	}()
+
+	// uptimeWindow specifies the duration in which host uptime is checked.
+	uptimeWindow = func() time.Duration {
+		switch build.Release {
+		case "dev":
+			return 24 * time.Hour // 1 day
```

Comment:
1 day is too long, 'dev' constants are what you use in the antfarm. We don't want to wait a day to realize that a host has gone offline if we are testing the handling of offline hosts.

Probably 30 minutes is better.